mqtt/mqtt-edgehub/tests/translation.rs

```diff
@@ -331,28 +325,21 @@ async fn ensure_subscribed(client: &mut TestClient) {
     client.subscriptions().recv().await;
 }
 
-fn start_server<N, Z>(broker: Broker<Z>, authenticator: N) -> ServerHandle
```

Comment:
Can we just re-use `server::start_server` ?

Reply:
we can't, it does not add message handler.